test/user-test.js

```diff
@@ -0,0 +1,12 @@
+import chai from 'chai';
+const expect = chai.expect;
+
+import { getTotalSpent } from "../src/user";
+import { sampleData } from "./sampleData";
+
+describe('getTotalSpent', () => {
+    const totalSpent = getTotalSpent(sampleData.currentUserBookings, sampleData.allRooms);
```

Comment:
What if the user bookings are 0? Does the function still return a number? Consider testing with an empty array of user bookings.

Reply:
good call!! Turns out it somehow works fine with no bookings created for a user, but added a test to cover that in case something changes in functionality down the line